lib/publiccloud/gce.pm

```diff
@@ -30,15 +30,7 @@ has username => sub { get_var('PUBLIC_CLOUD_USER', 'susetest') };
 sub init {
     my ($self, %params) = @_;
     $self->SUPER::init();
-    $self->provider_client(publiccloud::gcp_client->new(
-            region => $self->region,
```

Comment:
`region` is not defined in `gcp_client.pm`. Can we remove it from here?

Reply:
Take a look to lib/publiccloud/aws_client.pm
`has region => sub { get_var('PUBLIC_CLOUD_REGION', 'eu-central-1') };

Reply:
yes, I mean in `gcp_client.pm`. Maybe, it's just ok and `region` is not needed in GCP?

Reply:
Ah ok, sorry
it uses
`has gcr_zone => sub { get_var('PUBLIC_CLOUD_GCR_ZONE', 'eu.gcr.io') };